projects/onto-search/src/lib/onto-search-column-selector/onto-search-column-selector.component.scss

```diff
@@ -0,0 +1,23 @@
+.column-selector-component {
+  width: 100%;
+  height: 100%;
+  flex-direction: row;
+  box-sizing: border-box;
+  display: flex;
+  place-content: center flex-start;
+  align-items: center;
+
+  .column-select {
+    flex: 1 1 100%;
+    box-sizing: border-box;
+    max-width: 100%;
+  }
+
+  .form-field {
+    display: inherit;
+  }
+}
+
+button.column-reset-button {
```

Comment:
Move this selector inside `column-selector-component` and also remove the button tag selector.